### PR TITLE
refactor(lira): standardize output variable names

### LINKED ISSUE #444: Standardize naming for target outputs and attack signals across MIA attacks
Different MIA implementations currently use different names for similar values. Examples include:

- base: logits_target
- attack_p: attack_signal and audit_signal
- loss_trajectory: target
- LiRA and MS-LiRA: taget_model_logitsm, target_signals, shadow_models_signals, and sample_target_signals

Use the same naming pattern across all similar MIA attacks. 
The names should make it clear whether a variable contains the model’s original output or a signal calculated from that output. 
For example, use `target_outputs` and `shadow_outputs` or `target_model_outputs`  and `shadow_model_outputs`  for original model outputs.

It might be better with `outputs` than `logits`, because LeakPro also supports regression and forecasting models, which do not necessarily produce logits.

### PR DESCRIPTION
**Pull request by NiteeshGowda72**
## Summary

Standardize local variable names for raw model outputs in the LiRA attack.

## Changes

- Rename `target_logits` to `target_outputs`
- Rename `shadow_logits` to `shadow_outputs`
- Keep `target_signals` and `shadow_models_signals` unchanged
- Preserve the existing `load_logits()` API and attack behavior

## Testing

- `python3 -m py_compile leakpro/attacks/mia_attacks/lira.py`
- `python -m pytest leakpro/tests/mia_attacks/attacks/test_lira.py -v`
  - 7 passed
- `python -m pytest leakpro/tests/mia_attacks/attacks/test_multi_signal_lira.py -v`
  - 16 passed

Closes #444

### CODE REVIEW

**Review by fazelehh — CHANGES_REQUESTED** [review]
These changes only fix lira, but the issue is about all attacks.

**Comment by NiteeshGowda72** [comment]
Thanks for pointing that out. I initially scoped the change to LiRA based on the examples in the issue, but I understand now that the naming standardization should cover all relevant MIA attacks. I’ll update the other affected attack implementations as well and push the changes to this PR.

**Comment by NiteeshGowda72** [comment]
Hi, I have been reviewing the MIA implementations to standardize the naming consistently across the attacks. I have a clarification regarding QMIA, OSLO, and YOQO:

In QMIA, target_logits is populated from the configured self.signal(...), so it appears to represent a derived signal rather than raw model outputs.
In OSLO and YOQO, lprime initially contains raw target-model outputs and is then converted/overwritten into predictions.

Should these cases also be refactored as part of #444? In particular, should QMIA use a signal-oriented name, and should OSLO/YOQO separate the raw target_outputs from the resulting target_predictions?

I want to make sure the naming follows the intended convention before making the changes.

**Comment by NiteeshGowda72** [comment]
I’ve implemented the output naming standardization in the following files:

base.py
loss_trajectory.py
multi_signal_lira.py
rmia.py

The changes standardize names for raw model outputs while keeping calculated attack signals and existing APIs unchanged. I also ran the relevant tests, and all 46 tests passed.

I’ve intentionally left the following three implementations unchanged for now:

qmia.py — the existing target_logits values are produced through the configured signal, so I’d like to confirm whether these should be treated as attack signals rather than raw model outputs.
oslo.py — lprime is used in a way that appears to mix the model output and subsequent predictions, so I’d like clarification on the intended naming distinction.
yoqo.py — similarly, lprime is used for model output/prediction-related values, and I’d like to confirm the intended naming convention before changing it.

I left these three untouched pending clarification to avoid introducing a semantic change based on assumptions.

**Comment by NiteeshGowda72** [comment]
Hi @fazelehh , the changes are ready for review. Could you please take a look when you have time? Thanks!

**Comment by NiteeshGowda72** [comment]
Hi @fazelehh, I’ve addressed the requested changes and pushed the updated implementation. The remaining QMIA/OSLO/YOQO cases are left unchanged pending clarification as noted above. The PR is ready for review. Thank you!
